src/encoded/static/components/browse/components/AboveTableControls/SelectedFilesDownloadButton.js

```diff
@@ -0,0 +1,248 @@
+                                    For unpublished data sets, we ask that you please contact the data generating lab to discuss possible coordinated publication.
+                                    In your manuscript, please cite the 4DN White Paper (<a href="https://doi.org/10.1038/nature23884" target="_blank" rel="noopener noreferrer">doi:10.1038/nature23884</a>), and please acknowledge the 4DN lab which generated the data.
+                                    Please direct any questions to the <a href="mailto:support@4dnucleome.org">Data Coordination and Integration Center</a>.
+                                </li>
+                            : null }
+                        </ul>
+                    </div>
+
+                    { foundUnpublishedFiles && !disclaimerAccepted?
+                        <Button bsStyle="info" onClick={this.handleAcceptDisclaimer}>
+                            <i className="icon icon-fw icon-check mr-05"/>
+                            I have read and understand the notes.
+                        </Button>
+                        :
+                        <SelectedFilesDownloadStartButton {...{ selectedFiles, suggestedFilename }} />
+                    }
+
+                    <Button type="reset" onClick={onHide} className="ml-05">Cancel</Button>
+
+                </Modal.Body>
+            </Modal>
+        );
+    }
+}
+
+function ModalCodeSnippet(props){
+    const { filename, isSignedIn } = props;
+    return (
+        <pre className="mb-15">
+            cut -f 1 <b>{ filename }</b> | tail -n +3 | grep -v ^# | xargs -n 1 curl -O -L
+            { isSignedIn ? <code style={{ 'opacity' : 0.5 }}> --user <em>{'<access_key_id>:<access_key_secret>'}</em></code> : null }
+        </pre>
+    );
+}
+
+
+/**
+ * Use this button to download the tsv file metadata.
+ * Also has a close modal button.
+ */
+export class SelectedFilesDownloadStartButton extends React.PureComponent {
+
+    constructor(props){
+        super(props);
+        this.getAccessionTripleArrays = this.getAccessionTripleArrays.bind(this);
+    }
+
+    getAccessionTripleArrays(){
+        return _.map(
+            _.keys(this.props.selectedFiles),
+            function(accessionTripleString){
+                var accessions = accessionTripleString.split('~');
+                return [accessions[0] || 'NONE', accessions[1] || 'NONE', accessions[2] || 'NONE'];
+            }
+        );
+    }
+
+    /**
+    * This function renders out a literal form which may be submitted, with 'accession triples'
+    * ([ExpSetAccession, ExpAccession, FileAccession]) included in the POSTed form fields which
+    * identify the individual files to download.
+    *
+    * @returns {JSX.Element} A modal instance.
+    */
+    render(){
+        const { onClose, suggestedFilename } = this.props;
+        return (
+            <form method="POST" action="/metadata/?type=ExperimentSet&sort=accession" className="inline-block">
```

Comment:
Added `className="inline-block"` to form element to allow contents to be rendered in-line, e.g. side-by-side with other externally-placed buttons, removing need for Cancel button to be placed inside the form as well.